tools/trace-plotter/plotter/plotter.go

```diff
@@ -0,0 +1,198 @@
+package plotter
```

Comment:
hmm, so are we using go plotter or python plotter? please remove whatever is not used